release/models/bfd/openconfig-bfd.yang

```diff
@@ -138,48 +144,61 @@ module openconfig-bfd {
           "The BFD echo function failed - echo packets have not been
           received for the required period of time.";
       }
-      enum FORWARDING_RESET {
+      enum NEIGHBOR_SESSION_DOWN {
         value 3;
+        description
+          "The neighbor has signaled the session down.";
+      }
```

Comment:
Can you please add a blank like between each of these `enum` values? Not sure how we missed this originally. Thanks.

Reply:
Between each enum variant definition or between the value and description?

Looking at all other models, there are no additional lines between either ... Do you have an example in any models?

Reply:
Ah, apologies, yes - please ignore this. I had misparsed these as identities.